cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/opencv-control.tsx

```diff
@@ -365,8 +414,13 @@ class OpenCVControlComponent extends React.PureComponent<Props & DispatchToProps
                                     onInteractionStart(this.activeTool, activeLabelID);
                                     canvasInstance.interact({
                                         enabled: true,
+                                        onChangeBlockState: this.onChangeBlockState,
                                         ...this.activeTool.params.canvas,
                                     });
+                                    const { onSwitchBlockMode, toolsBlockerState } = this.props;
+                                    if (!toolsBlockerState.buttonVisible) {
+                                        onSwitchBlockMode({ buttonVisible: true, algorithmsLocked: false });
+                                    }
```

Comment:
Should be done in reducer when start interaction

Reply:
Done.